locale/en/docs/meta/topics/dependencies.md

```diff
@@ -39,13 +39,13 @@ asynchronously at the operating system level.
 
 - [Documentation](http://docs.libuv.org/)
 
-### http-parser
+### llhttp
 
-HTTP parsing is handled by a lightweight C library called http-parser. It is
+HTTP parsing is handled by a lightweight C library called llhttp. It is
```

Comment:
```suggestion
HTTP parsing is handled by a lightweight TypeScript and C library called llhttp. It is
```

You'll need to word-wrap to 80 chars if you accept this change.

Reply:
Thanks for reviewing! Fixed at 6be662d